test/formal/redundant_store_unrelated.py

```diff
@@ -0,0 +1,63 @@
+from z3 import Solver, Int, IntVal, unsat
+import sys
```

Comment:
```suggestion
import sys
from z3 import Solver, Int, IntVal, unsat
```
python linter apparently complains otherwise